src/Microsoft.Health.Dicom.Api/Features/Audit/AuditLogger.cs

```diff
@@ -78,8 +78,8 @@ public class AuditLogger : IAuditLogger
             AuditMessageFormat,
             auditAction,
             AuditEventType,
-            _securityConfiguration.Authentication?.Audience,
-            _securityConfiguration.Authentication?.Authority,
+            _securityConfiguration.AuthenticationSchemes.FirstOrDefault()?.Audience,
```

Comment:
Still need to figure out exactly how to get the correct audience & authority here